assignments/assignments_parser.go

```diff
@@ -63,14 +64,15 @@ func parseAssignments(dir string, courseID uint64) ([]*pb.Assignment, error) {
 				// or it will cause a database constraint violation (IDs must be unique)
 				// The Name field below is the folder name of the assignment.
 				assignment := &pb.Assignment{
-					CourseID:    courseID,
-					Deadline:    FixDeadline(newAssignment.Deadline),
-					Language:    strings.ToLower(newAssignment.Language),
-					Name:        filepath.Base(filepath.Dir(path)),
-					Order:       uint32(newAssignment.AssignmentID),
-					AutoApprove: newAssignment.AutoApprove,
-					ScoreLimit:  uint32(newAssignment.ScoreLimit),
-					IsGroupLab:  newAssignment.IsGroupLab,
+					CourseID:         courseID,
+					Deadline:         FixDeadline(newAssignment.Deadline),
+					Language:         strings.ToLower(newAssignment.Language),
```

Comment:
While I remember; we should consider to rename this field and corresponding field in the yaml file. It should really be named `template`. But question is if we should support both some how. I guess we should make an github issue or trello card.

Reply:
We already have a card about this. The plan was to do it when moving from yaml files to UI forms, where we could provide a dropdown menu with all languages that have a ready template.